jte-models/src/main/jte/statictemplates/kmain.jte

```diff
@@ -8,17 +8,14 @@
 @param Set<TemplateDescription> templates
 @param Iterable<String> imports
 @param ModelConfig modelConfig
-
 @file:Suppress("ktlint")
 package ${config.packageName()}
 
 import gg.jte.models.runtime.*
 import gg.jte.ContentType
-import gg.jte.TemplateOutput
-import gg.jte.html.HtmlInterceptor
```

Comment:
I'm not 100% sure about this removal.

There is a call to `kmethod` a few lines below:

https://github.com/casid/jte/blob/31fc79df1f1a0ae59fe1e977c350e6b1aa1c5654/jte-models/src/main/jte/statictemplates/kmain.jte#L24

And that template later declares this variable:

https://github.com/casid/jte/blob/31fc79df1f1a0ae59fe1e977c350e6b1aa1c5654/jte-models/src/main/jte/statictemplates/kmethod.jte#L8

That is then used by `StaticJteModel` in the next line:

https://github.com/casid/jte/blob/31fc79df1f1a0ae59fe1e977c350e6b1aa1c5654/jte-models/src/main/jte/statictemplates/kmethod.jte#L9

Reply:
Yup! I'd say it's needed. Thanks for catching that